pom.xml

```diff
@@ -36,12 +36,11 @@
   </scm>
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <localTeamServerUrl>http://localhost:19080/Contrast/api</localTeamServerUrl>
-    <versions.maven-source-plugin>3.0.1</versions.maven-source-plugin>
-    <versions.maven-javadoc-plugin>3.0.0</versions.maven-javadoc-plugin>
-    <versions.maven-gpg-plugin>1.6</versions.maven-gpg-plugin>
-    <versions.nexus-staging-maven-plugin>1.6.8</versions.nexus-staging-maven-plugin>
     <versions.junit-jupiter>5.7.1</versions.junit-jupiter>
+    <versions.auto-value>1.8.2</versions.auto-value>
+    <versions.lombok>1.18.18</versions.lombok>
```

Comment:
assuming lombok is used in the same capacity as AutoValue in the old code and form objects? We really should prefer autovalue imo, so this is a good start - just weird we have both for the time being

Reply:
Right, it's still used by the old code for more than just making value types. We'll leave it in while we transition. I got them to place nicely together 🙂